src/main/java/IsaacCharacterGeneratorApplication.java

```diff
@@ -1,15 +1,39 @@
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Random;
+import javax.sound.sampled.Line;
 
 public class IsaacCharacterGeneratorApplication {
     public static void main(String[] args) {
-        IsaacCharactersRepository isaacCharactersRepository = new IsaacCharactersRepository();
-        List<String> issacCharacters = isaacCharactersRepository.getCharacters();
+//        IsaacCharactersRepository isaacCharactersRepository = new IsaacCharactersRepository();
```

Comment:
Staraj się usuwać wykomentowany kod, jeżeli będzie taka potrzeba to możesz go przywołać z historii gita.